adminapp/src/pages/MemberDetailPage.jsx

```diff
@@ -472,7 +474,7 @@ function MessageDeliveries({ messageDeliveries }) {
       title="Message Deliveries"
       headers={["Id", "Created", "Sent", "Template", "To"]}
       rows={messageDeliveries}
-      keyRowAttr="id"
```

Comment:
This should not need to change- if it does, there's a but with `keyRowAttr`. Please check it out.

Reply:
Or get rid of `keyRowAttr`, but don't keep it around if it doesn't work.